Core/src/main/java/com/paypal/android/core/HttpMethod.kt

```diff
@@ -0,0 +1,6 @@
+package com.paypal.android.core
+
+enum class HttpMethod(val asString: String) {
```

Comment:
We could remove the `asString` field and use the `name` value built into Enums.

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-enum/name.html